src/sp/pl_session.cpp

```diff
@@ -374,35 +366,21 @@ namespace cubpl
   void
   session::clear_interrupt ()
   {
-    m_is_interrupted = false;
-    m_interrupt_id = NO_ERROR;
+    m_interrupt_id = 0;
```

Comment:
기존 코드가 더 의도가 명확한 것 같습니다.
```suggestion
    m_interrupt_id = NO_ERROR;
```

Reply:
원복하겠습니다.